### PR TITLE
community[minor]: Add Astra DB chat message history store

### PR DESCRIPTION
**Pull request by mfortman11**
Adds `AstraDBChatMessageHistory` as a message store

### CODE REVIEW

File: examples/src/memory/astradb.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import { BufferMemory } from "langchain/memory";
```
Comment (jacoblee93) [inline examples/src/memory/astradb.ts:1]: We're trying to do all new examples with LCEL - could you update this to be more like: 

https://github.com/langchain-ai/langchainjs/pull/4450/files#diff-65d27e9b05b811faae72376bd775edc281f6b37233130c14f38613089a580554R1

File: libs/langchain-community/src/stores/message/astradb.ts [hunk 2]
```diff
@@ -0,0 +1,116 @@
+import { BaseListChatMessageHistory } from "@langchain/core/chat_history";
```
Comment (jacoblee93) [inline libs/langchain-community/src/stores/message/astradb.ts:1]: Can you also add a separate entrypoint + mark it as optional to `langchain-community/langchain.config.js` and run `yarn build` in community?

https://github.com/langchain-ai/langchainjs/blob/main/CONTRIBUTING.md#adding-an-entrypoint

**Comment by jacoblee93** [comment]
Thanks! See comments.

**Comment by mfortman11** [comment]
@jacoblee93 thanks for the feedback should be all set now

**Comment by jacoblee93** [comment]
Looks great, thanks!

**Pull request merged**